src/Library/demos/Text Fields/main.js

```diff
@@ -58,22 +58,14 @@ const completion = new Gtk.EntryCompletion();
 entry_completion.completion = completion;
 
 const list_store = Gtk.ListStore.new([String]);
-var iter;
-const words = [
-"a",
-"app",
-"apple",
-"apples",
-"applets",
-"application"
-];
+const words = ["a", "app", "apple", "apples", "applets", "application"];
 words.forEach((word) => {
-  iter = list_store.append();
-  iter = list_store.set_value(iter, 0, word);
+  const iter = list_store.append();
+  list_store.set_value(iter, 0, word);
 });
 completion.model = list_store;
 
-completion.text_column = 0;
+completion.set_text_column(0);
```

Comment:
@AkshayWarrier this fixed it. Don't ask me why :shrug: 

I'm glad this whole thing is getting deprecated, the API is weird. But it'll do fine for a demo/example